package.json

```diff
@@ -55,7 +55,7 @@
   "devDependencies": {
     "ava": "~0.19.1",
     "coveralls": "~2.13.1",
-    "eslint": "~4.1.0",
+    "eslint": "~4.1.1",
```

Comment:
Yes, per https://github.com/eslint/eslint#semantic-versioning-policy:

> _"According to our policy, any minor update may report more errors than the previous release (ex: from a bug fix). As such, we recommend using the tilde (~) in package.json e.g. "eslint": "~3.1.0" to guarantee the results of your builds."_

stylelint is also adopting this same policy for the upcoming v8.